android/2_build_toolchain.sh

```diff
@@ -181,17 +112,17 @@ export PKG_CONFIG_PATH=$PLATFORM_PREFIX/lib/pkgconfig
 export PKG_CONFIG_LIBDIR=$PKG_CONFIG_PATH
 export TARGET_HOST="i686-linux-android"
 
-install_lib_png
-install_lib_freetype
-install_lib_pixman
-install_lib_ogg
-install_lib_vorbis
-install_lib_modplug
+install_lib libpng-1.6.23
+install_lib freetype-2.6.3 --with-harfbuzz=no --without-bzip2
+install_lib pixman-0.34.0
+install_lib libogg-1.3.2
+install_lib libvorbis-1.3.5
+install_lib libmodplug-0.8.8.5
+install_lib libsndfile-1.0.27
+install_lib speexdsp-1.2rc3 --enable-sse --disable-neon
```

Comment:
Uh thats cool didnt knew you can pass args this way withou "" and then just using $@